libs/platform/src/lib/components/button/button.component.html

```diff
@@ -1,6 +1,6 @@
 <button #fdButton fd-button [attr.id]="id" [attr.title]="title" [attr.name]="name" [attr.aria-label]="ariaLabel"
     [glyphPosition]="glyphPosition" [label]="label" [type]="type" [attr.value]="value"
-    [attr.aria-selected]="ariaSelected" [attr.aria-disabled]="ariaDisabled" [compact]="contentDensity !== 'cozy'"
+    [attr.aria-selected]="ariaSelected" [attr.aria-disabled]="ariaDisabled" [compact]="contentDensity ? contentDensity !== 'cozy' : ''"
```

Comment:
I think it's better to extract the logic to separate method for better readability, like:
```
[compact]="isCompact()"
```

Reply:
Created function as suggested.

Reply:
> I think it's better to extract the logic to separate method for better readability, like:
> 
> ```
> [compact]="isCompact()"
> ```

It is best to bind to avoid functions for data binding, and instead try to use properties or subscribe to observables with `observableName$ | async` because using functions results in the function being called every time a digest cycle is ran, which is slightly less performant than binding to a property.  It is some extra code in our library but it's better for our users in the long term.  Search through the core library for `$ | async"` to get an idea of how we prefer to handle this

Reply:
Removed function, reused contentDensity property, which gets value from core (common) contentDensityService (subscribes the changes already.)